api/api_ethereum.go

```diff
@@ -1003,7 +1000,7 @@ func newEthTransactionReceipt(header *types.Header, tx *types.Transaction, b Bac
 	}
 
 	if b.ChainConfig().IsEthTxTypeForkEnabled(new(big.Int).SetUint64(blockNumber)) {
```

Comment:
I think we can remove this if condition also!

Reply:
This if-condition is checking whether the block's type is ethTxType or not. Isn't this conditional necessary?